### PR TITLE
Initial version with "auto quit"

### PR DESCRIPTION
**Pull request by philipabbey**
Quit the application after a user settable period of time based on a timeout value from the settings.

@Someone0nEarth is this non-glance app of any interest to you?

### CODE REVIEW

**Comment by Someone0nEarth** [comment]
Hi!

This would prevent the battery drain issue. 👍

But on the UX level, having a widget/glance is much more comfy for me: Only to swipe down or up for starting. As an app, I would have to press a button on the watch (using my thumb and one finger) and then selecting the app for starting.

**Comment by philipabbey** [comment]
I think we can have both.

- Testing has shown the widget still gives me what I want, a standard application, and what you want.
- We then add the auto quit so the app also exits.
   - This should either live along side the auto close of the widget, or be disabled (timeout=0).
 
Perhaps this combination gives us the best of all worlds?

**Comment by Someone0nEarth** [comment]
That should work for me.

So, we add the timer you implemented and also change the type from "watch-app" to "widget" in the manifest.xml .

There is one "ugly" thing left in my dirt & quick widget/glance hack: When the user swipes back for quitting the widget, in the end, there will be a unnecessary "black screen" to swipe off. This is a circumstance of how this application views are currently implemented. I will try to change this later today or tomorrow and then we can see, if we will like it and go further to change the app to widget type.

**Comment by philipabbey** [comment]
Suggest we close this one off as an interim solution that should be compatible (or at least tolerable) with the widget proposal from @Someone0nEarth.

File: resources/settings/settings.xml [hunk 1]
```diff
@@ -60,4 +70,12 @@
       type="boolean"
     />
   </setting>
+
+<!--
```
Comment (JosephAbbey) [inline resources/settings/settings.xml:74]: It is probably not the best idea to leave stale code that does not work commented out.

**Review by JosephAbbey — CHANGES_REQUESTED** [review]

**Review by JosephAbbey — APPROVED** [review]

**Pull request merged**